spec/dummy/client/webpack.common.config.js

```diff
@@ -8,9 +8,7 @@ const assetLoaderRules = [
       loader: 'url-loader',
       options: {
         limit: urlFileSizeCutover,
-
-        // Leading slash is needed
-        name: '/images/[name]-[hash].[ext]',
+        name: 'images/[name]-[hash].[ext]',
```

Comment:
We removed the leading slash because we already appended it as a trailing slash in webpack.client.rails.build.config.js as explained.